__tests__/itinerary.test.js

```diff
@@ -0,0 +1,20 @@
+
+const Itinerary = require('../src/itinerary.js');
+const Port = require('../src/port.js');
+const Ship = require('../src/ship.js');
+
+describe('itinerary of ship', () => {
+    test('determines which port the ship docks', () => {
+
+        expect(new Itinerary()).toBeInstanceOf(Object);
+    });
+
+    test('it has ports', () => {
+        const leedsPort =  new Port('Leeds');
+        const greecePort = new Port('Greece');
+        
+        itinerary = new Itinerary([leedsPort, greecePort]);
```

Comment:
itinerary is not declared here, in the other files you have declared the variables with a let keyword, and you reassigned the value later on the tests.
In this test file you dont have itinerary declared, you are missing a const from here